app/frontend/src/views/form/Success.vue

```diff
@@ -3,12 +3,12 @@
     <FormViewer :submissionId="s" :readOnly="true" displayTitle>
       <template #alert="{ form }">
         <div class="mb-5">
-          <h1 class="mb-5">
+          <h1 class="mb-5" :class="{ 'dir-rtl': isRTL }">
             <v-icon large color="success">check_circle</v-icon>
             {{ $t('trans.sucess.sucessFormSubmissn') }}
           </h1>
           <div v-if="form.showSubmissionConfirmation">
-            <h3>
+            <h3 :class="{ 'dir-rtl': isRTL }">
               <span class="d-print-none">
                 {{ $t('trans.sucess.keepRecord') }}
               </span>
```

Comment:
Minor bug - on line 16 the `trans.sucess.confirmationId` translations all have a `:` at the end, so we end up with two `::`

![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/141ee9b5-3ac6-42eb-93f2-00549de1075a)